### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by morenol**
Add changelog body to eox-tenant in order to create a pypi relase.

### CODE REVIEW

**Comment by Squirrel18** [comment]
I think we should add the full changelog including all versions. What do you think? @morenol

**Comment by morenol** [comment]
> I think we should add the full changelog including all versions. What do you think? @morenol
I think that that would be ideal but I am not sure how easy is to add them and if the effort to add them is worth it

**Comment by Squirrel18** [comment]
I'm pretty sure it's not easy, but I think we should do it anyway because what is the point of adding a changelog file if we are not going to document the older releases (35 releases)? Does the incoming changelog addition have to contain all the features so far? Maybe we can add a changelog record for only major versions. @morenol

**Comment by morenol** [comment]
> I'm pretty sure it's not easy, but I think we should do it anyway because what is the point of adding a changelog file if we are not going to document the older releases (35 releases)? Does the incoming changelog addition have to contain all the features so far? Maybe we can add a changelog record for only major versions. @morenol

The point is  to start report the changes for the new versions since they will be on pypi

**Comment by morenol** [comment]
@Squirrel18 I tried to add to the changelog some of the changes for the previous version, I might need help to add the other ones.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+[1.3.0] - 2019-12-12
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Added
+_____
+
+* Support multitenancy in async process.
+
+
+
```
Comment (Squirrel18) [inline CHANGELOG.rst:74]: Unnecessary blank lines.

**Comment by Squirrel18** [comment]
Thanks @morenol  Maybe @felipemontoya or @andrey-canon could help add more changelog versions to the file.

**Pull request merged**